webpack.embedding-sdk.config.js

```diff
@@ -140,4 +143,20 @@ module.exports = env => {
         }),
     ].filter(Boolean),
   };
+
+  config.resolve.alias = {
```

Comment:
we can probably do something smarter here but I didn't want to import too many things from the other webpack config. Maybe we can have a shared folder we import things from for both of these files

Reply:
i.e. return a config based on EE vs OSS